src/badge/Badge.tsx

```diff
@@ -2,6 +2,8 @@ import React from 'react'
 import cc from 'classcat'
 import isEmpty from 'lodash.isempty'
 
+import { StyledBadge } from './Badge.style'
+
 export interface BadgeProps {
   readonly className?: string
   readonly children: string | JSX.Element | number
```

Comment:
`React.Node` is all of that

https://flow.org/en/docs/react/types/#toc-react-node
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a126f967c6e23e389e3cd9497132412ee857ee1e/types/react/index.d.ts#L239

Reply:
I'd rather keep the current definition, to keep the PR only about the new storybook